### PR TITLE
Add publishing workflow

### PR DESCRIPTION
**Pull request by VidarHUN**
Have to create these secrets: 
NPM_AUTH_TOKEN <npmjs user auth token>
DOCKER_USER <docker username>
DOCKER_TOKEN <docker token>

### CODE REVIEW

**Comment by VidarHUN** [comment]
After a new release everybody have to make a git pull.

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      - name: node
+        uses: actions/setup-node@v1.4.2
+        with:
+          node-version: 14
+          registry-url: https://registry.npmjs.org
+      - name: versioning
+        run: npm version --new-version "${GITHUB_REF:11}" --no-git-tag-version
+      - name: commit changes
+        run: |
+          git config --global user.email "robot@git.com"
```
Comment (nemethf) [inline .github/workflows/publish.yml:24]: better send it from a non-existent domain.  robot@l7mp.invalid?

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+          git config --global user.name "Git Robot"
+          git add .
+          git commit -m "Update package version via git"
```
Comment (nemethf) [inline .github/workflows/publish.yml:27]: "Sync package version with the latest git tag" ?

File: .github/workflows/publish.yml (hunk 1 above)
Comment (VidarHUN) [reply .github/workflows/publish.yml:24]: I will change.

File: .github/workflows/publish.yml (hunk 2 above)
Comment (VidarHUN) [reply .github/workflows/publish.yml:27]: Sounds better, tomorrow I fix it.

**Comment by VidarHUN** [comment]
Merging after consulting with @rg0now.

**Pull request merged**